registry/server/tag_cleanup.go

```diff
@@ -0,0 +1,82 @@
+package server
+
+import (
+	"context"
+	regexp2 "regexp"
+	"strconv"
+	"time"
+)
+
+var tagMarkTimeKey = "tagMarkedForDeletionTSMin"
+var allowedTagStalenessMinutes = 60 // todo: pass this through via config
+
+// todo: should I handle errors, or just log?
+// todo: what calls this function? A timer? Should anything happen if it fails?
```

Comment:
I think starting a background goroutine that also runs at some configurable interval and calls this would be a good idea. Re: the above question on error handling, I guess it depends what kind of error is returned. You can specify error types in here and conditionally handle them (ie whether certain types should be considered retryable or if you just log and continue).

Two examples for fun reference:
- The event writer in autothrottle (super simple, kind of dumb): task started [here](https://github.com/DataDog/kafka-kit/blob/c368b15888a0932fbbb1a5368f0a1d90fe1a51bc/cmd/autothrottle/main.go#L139), task contents [here](https://github.com/DataDog/kafka-kit/blob/c368b15888a0932fbbb1a5368f0a1d90fe1a51bc/cmd/autothrottle/events.go#L34)
- Registry listener, takes a context and handles task cancelation (this is preferable; any time you start a goroutine, you should know how it will exit and ideally have a way to control it): starts [here](https://github.com/DataDog/kafka-kit/blob/c368b15888a0932fbbb1a5368f0a1d90fe1a51bc/cmd/registry/main.go#L113), task contents [here](https://github.com/DataDog/kafka-kit/blob/c368b15888a0932fbbb1a5368f0a1d90fe1a51bc/registry/server/server.go#L103).